ts/components/definition.tsx

```diff
@@ -10,6 +10,7 @@ export interface Action {
     url?: string;
     onClick?: () => void;
     shouldUseAnchorTag?: boolean;
+    isSameTarget?: boolean;
```

Comment:
**TODO:** use `onClick`  and push new route then this is not needed

Reply:
fixed to use react-router properly :)